### PR TITLE
this fixes everything

### PR DESCRIPTION
**Pull request by erplsf**
because previously I made a typo in password generation nothing worked. now it does

### CODE REVIEW

**Review by mar-rih — APPROVED** [review]

File: pg_migration_tool/main.py [hunk 1]
```diff
@@ -214,10 +214,10 @@ def generate_pg_dump_and_restore_cmd(self, event: Select.Changed)-> str:
     @on(Button.Pressed)
     def button_pressed(self, event: Button.Pressed):
         if event.button.id == "migrate":
-            event.button.disabled = True
-            self.query_one(Select).disabled = True
+            self.query_one(Log).clear()
```
Comment (pawelros) [inline pg_migration_tool/main.py:217]: Will it not make it possible to accidentally click button multiple times and start multiple migrations at the same time?

**Review by pawelros — CHANGES_REQUESTED** [review]

File: pg_migration_tool/main.py (hunk 1 above)
Comment (erplsf) [reply pg_migration_tool/main.py:217]: it will. what's your suggestion? is there a way to wait until process started by thread finishes?

Comment (pawelros) [reply pg_migration_tool/main.py:217]: I would just disable button and restart the program after each migration. But I am ok with this as the chances of clicking twice are relatively small .

Comment (erplsf) [reply pg_migration_tool/main.py:217]: then I'll go the safe way and disable them anyway and restart from scratch.

Comment (erplsf) [reply pg_migration_tool/main.py]: @pawelros please review now

**Review by pawelros — APPROVED** [review]

**Pull request merged**